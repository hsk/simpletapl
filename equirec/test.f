/* Examples for testing */
lambda x:A. x;
lambda f:Rec X.A->A. lambda x:A. f x;

lambda x:T. x;
T;
i : T;
i;
