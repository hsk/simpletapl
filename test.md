# ここのところやってたこと

- PrologにTAPLのOCamlコードを移植。
- 移植したコードを更に短くした。
- 本質的な部分の取り出しを行った。
