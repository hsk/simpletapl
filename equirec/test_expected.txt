lambda x:A.x : A -> A
lambda f:Rec X.A -> A.lambda x:A.f x : (Rec X.A -> A) -> A -> A
lambda x:T.x : T -> T
T
i : T
i : T
